### PR TITLE
Serial number and multi device support features (plus optimizations)

### PR DESCRIPTION
**Pull request by neoben**
Add logic to handle specific device (or multiple devices) by serial number and add feature to bind csv file with specific device.
Please, find more detailed description in commit logs.

### CODE REVIEW

**Comment by ikstream** [comment]
That are a lot of changes, I will test it the next days.
Thank you for your contribution.

**Comment by neoben** [comment]
@ikstream Yes, I pushed a lot of changes.
I had to refactor a lot of functions in order to manage the device using a specific ID.
Anyway the logic is 100% compatible with the old behavior. 
Let me know if you have any doubt or questions.
Thanks!

**Comment by ikstream** [comment]
I am sorry, i had not much time to test your code, but i saw a few things so far. 
Please insert line break around 80 character.

File: README.md [hunk 1]
```diff
@@ -57,3 +57,42 @@ To create a sawtooth signal starting at 0dB increasing in 2dB steps every 50 mic
 "sudo attenuator_lab_brick -ramp -start 0 -end 60 -step 2 -t 50 us -rr 8"
 ```
 For a more enhanced usage example of our tool within your wireless experiments look at the "run-experiment.sh" shell script.
+
+## More (specific) usage examples
+Set 40dB attenuation to single device (automatically detected) for 20 seconds
+
+"sudo attenuator_lab_brick s -a 40 -t 20"
```
Comment (ikstream) [inline README.md]: The commands should be enclosed with code blocks as used in the rest of the README file (\`\`\`\n command \n\`\`\`)

Comment (neoben) [reply README.md]: Hi @ikstream,
you're right, I didn't pay attention to the README file (I just wrote some quick guidelines).
Do you want me to do that change or you are working on that?
I _allowed edits_ from maintainers so you are free to edit the code if you need so.

Comment (ikstream) [reply README.md]: Would be great if you could edit it by yourself

Comment (neoben) [reply README.md]: Ok, I've just pushed some changes to the README file.
Let me know if now it is ok or if you need some other changes.
Cheers!

**Review by ikstream — APPROVED** [review]

**Comment by ikstream** [comment]
@thuehn works for me
@neoben thank you

**Comment by neoben** [comment]
Glad to help and contribute guys!

**Comment by thuehn** [comment]
A big thank you for your effort NOEBEN !!!
Merged & ready for testing.

**Comment by neoben** [comment]
Great, thank you guys!

**Pull request merged**